docs/concepts/artifacts.md

```diff
@@ -178,53 +202,49 @@ if __name__ == "__main__":
     my_fn()
 ```
 
-As you can see, you don't need to create an artifact in a flow run context. You can use it however you like and still get the benefits in the Prefect UI.
+![Table artifact with customer info](/img/ui/table-artifact-info.png)
+As you can see, you don't need to create an artifact in a flow run context.
+You can use it however you like and still get the benefits in the Prefect UI.
 
-## Managing Artifacts
+## Managing artifacts
 
-### Reading Artifacts
+### Reading artifacts
 
-In the Prefect UI, you can view all of the latest versions of your artifacts and click into a specific artifact to see its lineage over time. Additionally, you can inspect all versions of an artifact with a given key by running:
+In the Prefect UI, you can view all of the latest versions of your artifacts and click into a specific artifact to see its lineage over time.
+Additionally, you can inspect all versions of an artifact with a given key by running:
 
-<div class="terminal">
 ```bash
-$ prefect artifact inspect <my-key>
+prefect artifact inspect my_key
 ```
-</div>
 
 or view all artifacts by running:
 
-<div class="terminal">
 ```bash
-$ prefect artifact ls
+prefect artifact ls
 ```
-</div>
 
 You can also use the [Prefect REST API](https://app.prefect.cloud/api/docs#tag/Artifacts/operation/read_artifacts_api_accounts__account_id__workspaces__workspace_id__artifacts_filter_post) to programmatically filter your results.
 
-### Deleting Artifacts
+### Deleting artifacts
 
 You can delete an artifact directly using the CLI to delete specific artifacts with a given key or id:
 
-<div class="terminal">
 ```bash
-$ prefect artifact delete <my-key>
+prefect artifact delete my_key
 ```
-</div>
 
-<div class="terminal">
 ```bash
-$ prefect artifact delete --id <my-id>
+prefect artifact delete --id my_id
```

Comment:
I don't believe there is a precedent on this page for using `my_key` or `my_id`, and I believe the convention when a user should pass in their own arg is to use the angle brackets. I'd prefer to keep them unless there is some reason to remove them.

Reply:
I think it's less cognitive overhead for the user to not have to think about what they need to replace when copying a code snippet.

Reply:
The format for user specified names of things is something that we should standardize in the style guide. Right now, it varies across our docs. I generally like an unadorned `my_the_thing` format, because I think any other characters make it unclear to the user whether they need those characters: e.g., `<>`, `[]`, `""`. That said, you rarely need the `<>` and I see docs with all those options. I'll do some research onthe  most common/best practice option and share it and get the style guide updated.